res/Sectors/M1105/Ricenden.txt

```diff
@@ -0,0 +1,466 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:11:53+00:00
+
+# Ricenden
+# 5,2
+
+# Name: Ricenden (hv)
+# Name: Gnoghikt! (kk)
+
+# Milieu: M1105
+
+# Credits: Ricenden sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Giostuxe
+# Subsector B: Nuana
+# Subsector C: Noonol
+# Subsector D: Osstaet
+# Subsector E: Etdirkorneer
+# Subsector F: Theeka
+# Subsector G: Phleskfan
+# Subsector H: Bluaskup
+# Subsector I: Hive
+# Subsector J: Kont
+# Subsector K: Iytganornim
+# Subsector L: Phlesige
+# Subsector M: Omeq
+# Subsector N: Lyarysklyixana
+# Subsector O: Luhawro
+# Subsector P: Eamlammaxaar
+
+# Alleg: Hf: "Hive FDA Uplift/Observation Site"
+# Alleg: HvFd: "Hive Federation"
+# Alleg: Kr: "Krotan Primacy"
+# Alleg: CsHv: "Hiver Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done.